File: src/main/java/no/nav/domain/pensjon/regler/krav/Kravlinje.java
package no.nav.domain.pensjon.regler.krav;

import no.nav.domain.pensjon.regler.PenPerson;
import no.nav.domain.pensjon.regler.kode.KravlinjeTypeCti;

import java.io.Serializable;
import java.lang.reflect.Field;

/**
 * En Kravlinje er en del av et KravHode. Eksempler på Kravlinje er GP, ET, UP
 * osv.
 */
public class Kravlinje implements Serializable {
    

    /**
     * Hvilken type kravlinjen gjelder, spesifisert som VilkarsvedtakType.
     */
    private KravlinjeTypeCti kravlinjeType;

    /**
     * Personen kravet relaterer seg til.
     */
    PenPerson relatertPerson;

}

File: src/main/java/no/nav/pensjon/regler/domain/krav/Kravlinje.kt
package no.nav.pensjon.regler.domain.krav

import no.nav.pensjon.regler.domain.PenPerson
import no.nav.pensjon.regler.domain.kode.KravlinjeTypeCti
import java.io.Serializable

/**
 * En Kravlinje er en del av et KravHode. Eksempler på Kravlinje er GP, ET, UP
 * osv.
 */
class Kravlinje : Serializable {
    /**
     * Hvilken type kravlinjen gjelder, spesifisert som VilkarsvedtakType.
     */
    var kravlinjeType: KravlinjeTypeCti? = null

    /**
     * Personen kravet relaterer seg til.
     */
    var relatertPerson: PenPerson? = null

}